### ISSUE TITLE
Unable to package CLI app that uses "image-size"

### ISSUE THREAD

**Issue by ZaLiTHkA** [zeit/pkg#80]
TLDR: I found Enclose, tried it and got an error. Found an issue report *over there* suggesting to use this instead. But I get the exact same message with Pkg.

> Note: For the record, I do actually want to replace `image-size` with something that lets me do true image comparisons (currently I'm using the MD5 file hash, which is getting messy).

My console output is as follows:

```
D:\dev-sandbox\spotlight-extractor>pkg -t node6-win index.js
> Fetching base Node.js binaries to: ~/.pkg-cache
  fetched-v6.10.0-win-x86      [====================] 100%
> Warning Cannot resolve ''./types/' + type'
  D:\dev-sandbox\spotlight-extractor\node_modules\image-size\lib\index.js
  Use a string literal as argument for 'require', or leave it
  as is and specify the resolved file name in 'scripts' option.
> Warning Cannot resolve ''./types/' + type'
  D:\dev-sandbox\spotlight-extractor\node_modules\image-size\lib\detector.js
  Use a string literal as argument for 'require', or leave it
  as is and specify the resolved file name in 'scripts' option.
```

I have a few other projects and ideas that I want to package for distribution, but they're not yet ready for that just yet.. So for the sake of understanding this issue I'll ask my question using this project as reference for now.

Obviously modifying the library that causes the problem is one option, but is that the best way to handle this situation? Is there anything else that can be done to work around this?

**Comment by brunobg** [comment]
Add this to your package.json:

```
"pkg": {
    "scripts": [
      "./node_modules/image-size/lib/types/*.js"
     ]
}
```

**Comment by ZaLiTHkA** [comment]
Ah, I didn't quite understand what that what intended for from the project's readme file, not exactly the clearest explanation there.. I also needed to add the `bin` entry to `package.json` and change the source file in my `pkg` command.

I still get the same warning now, so at first it doesn't look any different, but the generated executable now runs properly. 👍 Thanks.

**Comment by igorklopov** [comment]
Reopening to fix warning eventually

**Comment by igorklopov** [comment]
fixed in 3d87aca8db664b3bc2e52b1be71f6824e0afc79a
